### PR TITLE
Added live typing and docs to test slurk locally

### PR DESCRIPTION
**Pull request by pkhdipraja**

### CODE REVIEW

File: docs/slurk_bots.rst [hunk 1]
```diff
@@ -236,3 +236,16 @@ can listen to a command with ``on_command``:
...
 - ``room``: the id of the room, where the command was entered, or ``None``
 - ``timestamp``
+
+
+Testing bots
+~~~~~~~~~~~~
+
+You can also test your custom bots locally using ``docker``. First, you should create a ``Dockerfile``
+inside the folder where your bots reside. The content of the ``Dockerfile`` should follow the one available
+under ``clp-research/slurk-bots``. Afterwards, you can build the docker container using
```
Comment (janagoetze) [inline docs/slurk_bots.rst:246]: let's add the exact name of the file

**Review by janagoetze — COMMENTED** [review]
this PR contains a lot more than just the documentation. let's separate the content. You can do it by creating a new branch from `development` and then use `git cherry-pick <commit>` using the relevant commit id (which you can copy from `git log`) and then open a PR from that branch instead.

File: docs/slurk_bots.rst (hunk 1 above)
Comment (pkhdipraja) [reply docs/slurk_bots.rst]: I already updated it with the exact filename

**Comment by pkhdipraja** [comment]
I've already created a new PR for the livetyping mode https://github.com/clp-research/slurk/pull/103

**Pull request closed without merging**